WooCommerce/Classes/ViewRelated/ReusableViews/TitleAndSubtitleTableViewCell.swift

```diff
@@ -0,0 +1,46 @@
+import UIKit
+
+final class TitleAndSubtitleTableViewCell: UITableViewCell {
+
+    @IBOutlet var subtitleLabel: UILabel!
+    @IBOutlet var titleLabel: UILabel!
+    @IBOutlet var statusContainerView: UIView!
+    @IBOutlet var statusLabel: PaddedLabel!
+
+    static func register(for tableView: UITableView) {
+        tableView.registerNib(for: self)
+    }
+
+    override func awakeFromNib() {
+        super.awakeFromNib()
+        configureBackground()
+        configureLabels()
+    }
+
+    override func setSelected(_ selected: Bool, animated: Bool) {
+        super.setSelected(selected, animated: animated)
+    }
```

Comment:
nit: this override doesn't seem to be implemented, maybe can be deleted?

Reply:
Deleted in 2fa7ddb1466f9b6f5943b935e9df144e76a44b97 👍